### DISCUSSION TITLE
Problem accessing a private repository while being a collaborator

### DISCUSSION THREAD

**Discussion by IKdotShark** [community/community#116685] [Projects and Issues]
### Select Topic Area

Bug

### Feature Area

Projects

### Body

Hi all. A problem arose, my colleague created a private repository and invited us - his colleagues. As I understand it, a question should appear in the notifications for the user who was offered co-authorship for this repository. My colleagues received such a message and they successfully accepted it. But I didn't get it.
What I tried to do:
* Relog in account
* Restart browser
* Person who is the creater (owner) of repo retry to add me as a contributor
* Owner recreated repo
* Checked settings of notification, but Idk mb something is wrong in it
Please I need help. Be glad for answer.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by jonasfluegge** [comment]
Have you already tried simply having the URL of the repo sent to you and opening it? The link may then redirect you to the invitation, etc.

**Reply by IKdotShark** [reply]
If u think what was the problem, idk, I just wait over a week, and problem disapiard)

But thx for try
